query-node/mappings/mappingsContent.ts

```diff
@@ -0,0 +1,464 @@
+
+  if (type == ProtobufEntity.Channel) {
+    return ChannelMetadata.deserializeBinary(metadata)
+    /*
+    return {
+      title: 'TODO handle', // TODO: read from protobuf
+      description: 'TODO description', // TODO: read from protobuf
+      coverPhoto: undefined, // TODO: read from protobuf
+      avatarPhoto: undefined, // TODO: read from protobuf
+      isPublic: true, // TODO: read from protobuf
+      language: undefined, // TODO: read language from protobuf and connect it with existing Language (if any)
+    }
+    */
+  }
+
+  if (type == ProtobufEntity.ChannelCategory) {
+    return ChannelCategoryMetadata.deserializeBinary(metadata)
+  }
+
+  if (type == ProtobufEntity.Video) {
+    return VideoMetadata.deserializeBinary(metadata)
+  }
+
+  if (type == ProtobufEntity.VideoCategory) {
+    return VideoCategoryMetadata.deserializeBinary(metadata)
+  }
+
+  throw `Not implemented type: ${type}`
+}
+
+// temporary function used before proper block is retrieved
+function convertblockNumberToBlock(block: number): Block {
+  return new Block({
+    block: block,
+    executedAt: new Date(), // TODO get real block execution time
+    network: currentNetwork,
+  })
+}
+
+/////////////////// Channel ////////////////////////////////////////////////////
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export async function content_ChannelCreated(db: DatabaseManager, event: SubstrateEvent): Promise<void> {
+  /* event arguments
+  ChannelId,
+  ChannelOwner<MemberId, CuratorGroupId, DAOId>,
+  Vec<NewAsset>,
+  ChannelCreationParameters<ContentParameters>,
+  */
+
+  const protobufContent = readProtobuf(ProtobufEntity.Channel, (event.params[3] as any).meta) // TODO: get rid of `any` typecast
+
+  const channel = new Channel({
+    id: event.params[0].toString(), // ChannelId
+    isCensored: false,
+    videos: [],
+    happenedIn: convertblockNumberToBlock(event.blockNumber),
+    ...protobufContent.toObject()
+  })
+
+  await db.save<Channel>(channel)
+}
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export async function content_ChannelUpdated(
+  db: DatabaseManager,
+  event: SubstrateEvent
+) {
+  /* event arguments
+  ContentActor,
+  ChannelId,
+  Channel,
+  ChannelUpdateParameters<ContentParameters, AccountId>,
+  */
+
+  const channelId = event.params[1].toString()
+  const channel = await db.get(Channel, { where: { id: channelId } })
+
+  const protobufContent = readProtobuf(ProtobufEntity.Channel, (event.params[3] as any).meta) // TODO: get rid of `any` typecast
+
+  const updatedChannel = new Channel({
+    // TODO: check that this kind of new updated channel construction is valid or it should rather be edited as `channel.myProperty = something`
+    ...channel,
+    ...protobufContent.toObject()
+  })
+
+  await db.save<Channel>(updatedChannel)
+}
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export async function content_ChannelDeleted(
```

Comment:
ChannelDeleted event has been dropped, I think you already mentioned that some events are now removed in latest runtime.

Reply:
Event handler has been removed.